components/brave_wallet/browser/brave_wallet_constants.cc

```diff
@@ -250,6 +251,265 @@ const std::vector<mojom::BlockchainToken>& GetSardineBuyTokens() {
   return *tokens;
 }
 
+const std::vector<mojom::BlockchainToken>& GetTransakBuyTokens() {
+  static base::NoDestructor<std::vector<mojom::BlockchainToken>> tokens(
+      {
```

Comment:
Nit that's best addressed in a separate issue - I'm thinking we're doing this commonly enough that it makes sense for us to abstract these lists into a separate repository that we can update with a component updater. Cc @yrliou whats your take on this?

Reply:
@kdenhartog We are already exploring this.